### PR TITLE
Issue551 sphinx spews many warnings

### LINKED ISSUE #551: Sphinx spews many warnings:
- [x]  `WARNING: sphinx.ext.pngmath has been deprecated. Please use sphinx.ext.imgmath instead.`
- [x]  `InputError:** [Errno 2] No such file or directory: 'documentation/GIT.txt'.`
- [x]  `/Users/guyer/Documents/research/FiPy/fipy/INSTALLATION.rst:571: WARNING: Unknown target name: "virtualenv".`
- [x]  `/Users/guyer/Documents/research/FiPy/fipy/INSTALLATION.rst:571: WARNING: Unknown target name: "virtualenv".`
- [x]  `/Users/guyer/Documents/research/FiPy/fipy/documentation/API.rst:8: WARNING: toctree contains reference to nonexisting document u'fipy/generated/fipy.models'`
- [x]  `/Users/guyer/Documents/research/FiPy/fipy/documentation/API.rst:8: WARNING: toctree contains reference to nonexisting document u'fipy/generated/fipy.test'`
- [x] `ImportError: cannot import name cylindricalNonUniformGrid1D`
- [x] `ImportError: No module named pyamg`
- [x] many other import errors
- [x] `ValueError: API 'QDate' has already been set to version 1`
- [x] `WARNING: duplicate label for keys ...`
- [x] `WARNING: document isn't included in any toctree`
- [x] `WARNING: Citation [...] is not referenced.`
- [x] `WARNING: unknown option:`
- [x] `WARNING: ... reference target not found: ...`
- [x] `WARNING: equation not found:`

### PR DESCRIPTION
**Pull request by guyer**
Fixes #551

### CODE REVIEW

File: examples/levelSet/electroChem/simpleTrenchSystem.py [hunk 1]
```diff
@@ -51,7 +51,7 @@
 for modeling electrodeposition using the CEAC mechanism. The
 material properties and experimental parameters used are roughly
 those that have been previously
-published :cite:`NIST:damascene:2003]`.
+published :cite:`NIST:damascene:2003`.
 
 To run this example from the base fipy directory type::
```
[the remark this answers is not in the document]
Comment (guyer) [reply examples/levelSet/electroChem/simpleTrenchSystem.py]: Nice catch!

File: documentation/bogus.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
+External Command-line Flags
```
[the remark this answers is not in the document]
Comment (guyer) [reply documentation/bogus.rst]: If we don't define them, we get noisy warnings

File: documentation/contents.rst [hunk 3]
```diff
@@ -24,6 +24,8 @@ FiPy Contents
    :hidden:
 
    CODESPEED
-   BUILDBOT
    GUIDELINES
    GIT
+   tutorial/package/generated/package
+   ../fipy/generated/fipy
+   bogus
```
[the remark this answers is not in the document]
Comment (guyer) [reply documentation/contents.rst]: killjoy

File: examples/phase/binaryCoupled.py [hunk 4]
```diff
@@ -538,15 +538,15 @@ def deltaChemPot(phase, C, T):
 field through ``phaseTransformationVelocity``, it is necessary sweep this
 non-linear problem to convergence. We use the "residual" of the equations
 (a measure of how well they think they have solved the given set of linear
-equations) as a test for how long to sweep. Because of the
-:class:`~fipy.terms.convectionTerm.ConvectionTerm`, the solution matrix for ``diffusionEq`` is asymmetric
-and cannot be solved by the default :class:`~fipy.solvers.LinearPCGSolver`. Therefore, we use a
+equations) as a test for how long to sweep. The solution matrix for ``diffusionEq`` is asymmetric
+(is this still true?)
```
[the remark this answers is not in the document]
Comment (guyer) [reply examples/phase/binaryCoupled.py]: Agreed. I've removed the comment and the explicit use of `LinearLUSolver`

**Comment by tkphd** [comment]
Very few warnings when building HTML, and a manageable number for LaTeX, with nothing undefined.

The first LaTeX build warns,
```
WARNING: invalid reference in examples.phase.binaryCoupled docstring
```
... but that does not appear to affect the output.

Looks good to me.

**Pull request merged**